.travis.yml

```diff
@@ -8,4 +8,6 @@ before_install:
   - npm install -g grunt@0.4.5
   - npm install -g grunt-cli
   - npm install -g bower
+  - export PHANTOMJS_CDNURL=https://bitbucket.org/ariya/phantomjs/downloads npm install phantomjs
+  - npm install -g phantomjs
```

Comment:
I think it is necessary based on this [advice](http://stackoverflow.com/a/23967945).